modules/utxo-lib/src/bitgo/UtxoTransactionBuilder.ts

```diff
@@ -25,14 +26,29 @@ export class UtxoTransactionBuilder<T extends UtxoTransaction = UtxoTransaction>
     if (txb) {
       (this as any).__INPUTS = (txb as any).__INPUTS;
     }
+
+    if (prevOutputs) {
+      const txbInputs = (this as any).__INPUTS;
+      if (prevOutputs.length !== txbInputs.length) {
+        throw new Error(`prevOuts must match txbInput length`);
+      }
+      prevOutputs.forEach((o, i) => {
+        txbInputs[i].value = o.value;
+        txbInputs[i].prevOutScript = o.prevOutScript;
+      });
+    }
   }
 
   createInitialTransaction(network: Network, tx?: Transaction): UtxoTransaction {
     return new UtxoTransaction(network, tx);
   }
 
-  static fromTransaction(tx: UtxoTransaction): UtxoTransactionBuilder {
-    return new UtxoTransactionBuilder(tx.network, TransactionBuilder.fromTransaction(tx));
+  static fromTransaction(
+    tx: UtxoTransaction,
+    network?: bitcoinjs.Network,
+    prevOutputs?: PrevOutput[]
+  ): UtxoTransactionBuilder {
+    return new UtxoTransactionBuilder(tx.network, TransactionBuilder.fromTransaction(tx), prevOutputs);
```

Comment:
I needed to add `network?: bitcoinjs.Network` here otherwise TS yells at me for not extending `bitcoinjs.TransactionBuilder` correctly